DEFLATE (ZLIB) compression
This option enables DEFLATE (also known as ZLIB) compression
algorithm support.
Depends: PSTORE
---
--- /var/tmp/portage/sys-kernel/gentoo-sources-5.19.17/image/usr/src/linux-5.19.17-gentoo//.config.orig	2022-10-31 13:57:59.705695527 -0400
+++ /var/tmp/portage/sys-kernel/gentoo-sources-5.19.17/image/usr/src/linux-5.19.17-gentoo//.config	2022-11-01 16:19:05.114610383 -0400
@@ -9331,19 +9331,18 @@
 CONFIG_ROMFS_ON_BLOCK=y
 CONFIG_PSTORE=y
 CONFIG_PSTORE_DEFAULT_KMSG_BYTES=10240
-CONFIG_PSTORE_DEFLATE_COMPRESS=y
+# CONFIG_PSTORE_DEFLATE_COMPRESS is not set
 CONFIG_PSTORE_LZO_COMPRESS=m
 CONFIG_PSTORE_LZ4_COMPRESS=m
 CONFIG_PSTORE_LZ4HC_COMPRESS=m
 CONFIG_PSTORE_842_COMPRESS=y
 # CONFIG_PSTORE_ZSTD_COMPRESS is not set
 CONFIG_PSTORE_COMPRESS=y
-CONFIG_PSTORE_DEFLATE_COMPRESS_DEFAULT=y
-# CONFIG_PSTORE_LZO_COMPRESS_DEFAULT is not set
+CONFIG_PSTORE_LZO_COMPRESS_DEFAULT=y
 # CONFIG_PSTORE_LZ4_COMPRESS_DEFAULT is not set
 # CONFIG_PSTORE_LZ4HC_COMPRESS_DEFAULT is not set
 # CONFIG_PSTORE_842_COMPRESS_DEFAULT is not set
-CONFIG_PSTORE_COMPRESS_DEFAULT="deflate"
+CONFIG_PSTORE_COMPRESS_DEFAULT="lzo"
 # CONFIG_PSTORE_CONSOLE is not set
 # CONFIG_PSTORE_PMSG is not set
 # CONFIG_PSTORE_FTRACE is not set
